api/Selection.json

```diff
@@ -1321,7 +1321,7 @@
               "version_added": "57"
             },
             "ie": {
-              "version_added": true
+              "version_added": false
```

Comment:
http://mdn-bcd-collector.appspot.com/tests/api/Selection uses an instance, so we can trust this true&rarr;false change.